### PR TITLE
Process multiple instances with different field types correctly in SchemaGenerator

### LINKED ISSUE #60: SchemaGenerator generates invalid Schema
Schema Generator identifies coords as Strings which should be doubles and therefore parsing the json file with the generated scehma fails. 

Instead of
 `GPScoords: Option[List[Option[List[Option[String]]]]]` 
I would expect
 `GPScoords: Option[List[Option[List[Option[Double]]]]]`

image

@johanneshiry would you mind to have a short look at this? 🙄 I will push a branch with the state where this occurs.

### PR DESCRIPTION
**Pull request by johanneshiry**
Resolves #60

Please carefully execute some tests on your local machine when reviewing this PR. Even though all tests are passing + I added new test I'm not 100% that I covered all corner cases. Hence, please try it with several cases on your local machine (schema generation + parsing of some grid files) on order to ensure valid functionality. Furthermore, if you have any other corner cases please consider adding some tests accordingly.

### CODE REVIEW

File: src/main/scala/edu/ie3/powerFactory2psdm/util/SchemaGenerator.scala [hunk 1]
```diff
@@ -19,7 +19,7 @@ object SchemaGenerator extends LazyLogging {
   def main(args: Array[String]): Unit = {
     val source =
       Source.fromFile(
-        s"${new File(".").getCanonicalPath}/src/main/python/pfGridExport/pfGrid.json"
+        s"${new File(".").getCanonicalPath}/src/test/resources/pfGrids/exampleGrid.json"
```
Comment (t-ober) [inline src/main/scala/edu/ie3/powerFactory2psdm/util/SchemaGenerator.scala:22]:
```suggestion
        s"${new File(".").getCanonicalPath}/src/main/python/pfGridExport/pfGrid.json"
```

**Review by t-ober — CHANGES_REQUESTED** [review]

**Review by t-ober — APPROVED** [review]
